docs/tutorial/moving-from-electron.md

```diff
@@ -0,0 +1,135 @@
+# Moving from Electron to Muon
+
+Muon is a fork of Electron with improved security and support for Chrome extensions.
+
+While both provide similar features, it's important to note that Muon has mostly diverged from Electron changes since around Electron v1.4.13 (Dec 20, 2016) due to architectural differences [[1]](#ref1).
+
+This document details API differences which affect switching to Muon.
+
+As Muon is actively maintained for use in [Brave Browser](https://brave.com/), it can also be useful to study the code found in [brave/browser-laptop](https://github.com/brave/browser-laptop).
+
+## Prebuilt binaries
+
+Prebuilt binaries for Muon are provided by the [brave/electron-prebuilt](https://github.com/brave/electron-prebuilt) repo. It can be used in `package.json` along with related variables configured in `.npmrc`. Running `npm install && npm rebuild` after adding the following changes should switch your project to use Muon.
+
+`package.json`
+```json
+{
+	"devDependencies": {
+		"electron": "brave/electron-prebuild"
```

Comment:
should be `brave/electron-prebuilt`

Reply:
Good catch, fixed

Reply:
It doesn't have to be tho.

I've found the usage of 'electron' rather than 'electron-prebuild' helps with packaging through electron-builder.